shared/Docs/Layout.tsx

```diff
@@ -15,7 +15,8 @@ import { SectionProvider } from "./SectionProvider";
 import { useMobileNavigationStore } from "./MobileNavigation";
 import { getOpenGraphImageURL } from "../../utils/social";
 
-const GITHUB_PREFIX = "https://github.com/inngest/website/tree/main/";
+const GITHUB_BRANCH = process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF || "main";
```

Comment:
Tested on Vercel preview - this works correctly on the branch build. We'll have to test after this PR is merged that this is correct on production. `NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF` is supposed to be set to `main` on production, but we should confirm this when out.